opentech/apply/funds/tables.py

```diff
@@ -115,6 +115,14 @@ class Meta(BaseAdminSubmissionsTable.Meta):
         orderable = False
 
 
+class SummarySubmissionsTableWithRole(BaseAdminSubmissionsTable):
+    """ Adds Role Assigned to the 'Waiting for My Review' table """
+    role_assigned = tables.Column(verbose_name="Assigned Role")
+
+    class Meta(BaseAdminSubmissionsTable.Meta):
+        orderable = False
+
```

Comment:
You need a way to render the icon and i think we need to tweak the queryset method to add in the role.

so here:
```
from wagtail.images.views.serve import generate_image_url

def render_role_assigned(record, request):
    return  '<img src="' + generate_image_url(record.role_icon, 'max-12x12') + '" />'
```
I think. Or we could use the `ImageNode` from `wagtail.images.templatetags.wagtail_images` to return a node which will be rendered (`ImageNode(record.role_icon, 'max-12x12')` but don't quote me on that)